fmlearlydisplay/src/main/java/net/minecraftforge/fml/earlydisplay/RenderElement.java

```diff
@@ -84,16 +84,6 @@ private static void startupLogMessages(SimpleBufferBuilder bb, SimpleFont font,
 
         font.generateVerticesForTexts(10, context.scaledHeight() -  texts.size() * font.lineSpacing() + font.descent() - 10, bb, texts.toArray(SimpleFont.DisplayText[]::new));
     }
-    public static RenderElement monag() {
-        return new RenderElement(RenderElement.initializeTexture("monagstudios.png", 45000, 4, (bb, ctx, sz, frame) -> {
-            var size = 256;
-            var x0 = (ctx.width() - 2 * size) / 2;
-            var y0 = 64;
-            QuadHelper.loadQuad(bb, x0, x0+size, y0, y0+size/2f, 0f, 1f, 0f, 0.5f, 0xFFFFFFFF);
-            QuadHelper.loadQuad(bb, x0+size, x0+2*size, y0, y0+size/2f, 0f, 1f, 0.5f, 1f, 0xFFFFFFFF);
-        }));
-
-    }
```

Comment:
Good riddance. Though, I think it'd be cool if some future implementation of the early display window could support extensions such as this.

As a reminder, in the future the best way to re-implement this would be a **separate window** from Minecraft that does not run on the render thread.

cc @basdxz

Reply:
This just deletes a unused meme things that was added for some dumb reason. Not sure what you're talking about.

Reply:
For example, if a mod or modpack wants to add, replace, or remove images or text from the early display window. It's not something that would be necessary but is something that modders already do with hacks.

Reply:
Its completely outside the scope of this pr and has nothing to do with the current code. So not the place.
And honestly I  would rather just delete the entire early loading screen because of how hacky it is.
But feel free to make a feature request in a different PR if you think it is something people want.

Reply:
I'll talk to Ven about it some time next year. Been thinking about it a decent amount.